app-crypt/argon2/argon2-20171227.recipe

```diff
@@ -0,0 +1,100 @@
+SUMMARY="A password-hashing function"
+DESCRIPTION="Argon2 is a password-hashing function that summarizes the state \
+of the art in the design of memory-hard functions and can be used to hash \
+passwords for credential storage, key derivation, or other applications."
+HOMEPAGE="https://github.com/p-h-c/phc-winner-argon2"
+COPYRIGHT="2015-2017 Daniel Dinu
+	Dmitry Khoravich
+	Jean-Philippe Aumasson
+	Samuel Neves."
+LICENSE="Apache v2
+	CCO 1.0"
+REVISION="1"
+SOURCE_URI="https://github.com/P-H-C/phc-winner-argon2/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="eaea0172c1f4ee4550d1b6c9ce01aab8d1ab66b4207776aa67991eb5872fdcd8"
+SOURCE_DIR="phc-winner-argon2-$portVersion"
+PATCHES="argon2-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	argon2$secondaryArchSuffix = $portVersion
+	cmd:argon2$secondaryArchSuffix = $portVersion
+	lib:libargon2$secondaryArchSuffix = 1 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	argon2${secondaryArchSuffix}_devel = $portVersion
+	devel:libargon2$secondaryArchSuffix  = 1 compat >= 1
+	"
+REQUIRES_devel="
+	argon2$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage argon2$secondaryArchSuffix \
+	$libDir/libargon2.so.1
+
+BUILD()
+{
+	make PREFIX=$prefix
+}
+
+TEST_REQUIRES="
+	cmd:diff
+	"
+
+INSTALL()
+{
+	make install PREFIX=$prefix
```

Comment:
```suggestion
	make install PREFIX=$prefix INST_INCLUDE=$includeDir INST_LIBRARY=$libDir INST_BINARY=$binDir
```

One line to solve pretty much all problems after ;)

Reply:
Nice catch, looked into the Makefile, but missed this line (looked into the lines before these) :)